patches/chrome-installer-linux-rpm-build.sh.patch

```diff
@@ -26,8 +26,11 @@ index 0bcd8689d45850ad539f99423fa211785db4f343..cec0813f491a570fb499e7de5adb9d13
    gen_spec
  
    # Create temporary rpmbuild dirs.
-@@ -108,7 +115,10 @@ do_package() {
+@@ -106,9 +113,13 @@ do_package() {
+   log_cmd fakeroot rpmbuild -bb --target="$ARCHITECTURE" --rmspec \
+     --define "_topdir $RPMBUILD_DIR" \
      --define "${COMPRESSION_OPT}" \
++    --define "_buildhost brave-build" \
```

Comment:
I'd really like to avoid patching if there are other options - "We can change the hostname in the user data when the instance is being launched"